src/studio/src/designer/frontend/packages/schema-editor/src/utils.ts

```diff
@@ -28,12 +28,14 @@ export function getUiSchemaItem(schema: UiSchemaItem[], path: string): UiSchemaI
   return result;
 }
 
-export const getParentPath = (path: string): string | null => {
+export const getParentPath = (path: string): [string | null, string | null] => {
```

Comment:
the function name here is a bit confusing.
It seems as though the function is returning a path-and-name.
maybe `splitParentPathAndName` is a bit more descriptive?